README.md

```diff
@@ -2,24 +2,47 @@
 
 # Databraid Slack API
 
-## Usage
 
-Before beginning make sure you have [Docker](https://www.docker.com/) installed and running, then do:
+## Description
 
-```
-npm i
-npm run up
-```
+An Express server/router which serves as the backend for the Databraid Slack Widget.  This is used in conjunction with the Databraid Slack SPA.  It is built to run as a container within Docker.
 
-When finished developing do:
+The Databraid Slack API handles all interaction with Slack's Event API.  It authenticates with Slack during an OAuth sign-in.  It makes requests to the Slack Event API to fetch information about a Slack team's users and channels.  It interfaces with a PostgreSQL database to store and retrieve data. It interfaces with Google's Natural Language Processor API to perform sentiment analysis on the channel's content. It is also responsible for setting up a web socket and, once the Databraid Slack SPA connects with that socket, pushing new event and sentiment information to the client.
```

Comment:
@Dave-Gallup if there is a quick link to the Slack Event API, please provide it

Reply:
Done